viz/viz.js

```diff
@@ -2,125 +2,133 @@
 
 import earcut, {flatten, deviation} from '../src/earcut.js';
 
-const testPoints = [
```

Comment:
I added while debugging to make it easy to switch between test fixtures, for example http://localhost:8000/viz/?fixture=touching-holes5&rotation=180, but can roll it back if you don't want it.